src/utils/websockets.js

```diff
@@ -34,10 +38,33 @@ export function connectWS(topic, onMessageCallback) {
   };
 
   const connect = () => {
-    this.sock = new SockJs(config.WS_URL);
-    this.sockClient = Stomp.Stomp.over(this.sock);
-    this.sockClient.debug = null;
-    this.sockClient.connect({}, subscribe);
+    this.sockClient = new Client({
+      brokerURL: config.WS_URL,
+      // debug: function(str) {
+      //   console.log('debug: ', str);
+      // },
+      reconnectDelay: 5000,
+      heartbeatIncoming: 4000,
+      heartbeatOutgoing: 4000,
+    });
+
+    if (process.env.JEST_WORKER_ID === undefined) {
```

Comment:
quotes around undefined

Reply:
it's not `typeof` so it won't be a string